devtools/cli/distribution/jreleaser.yml

```diff
@@ -0,0 +1,61 @@
+project:
+  name: quarkus
+  description: CLI for Quarkus
+  longDescription: |
+    Create projects, manage extensions and perform build and development tasks for Quarkus,
+    the Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM.
+  website: https://quarkus.io
+  authors:
+    - Quarkus Community
+  license: Apache-2.0
+  # TODO: check with Max what we want to do here
+  copyright: Red Hat and other contributors
```

Comment:
@maxandersen can you make sure this is what we want here? Or something like `Quarkus contributors`?

Reply:
Lets start with "Quarkus Contributors". 

We Can make it more specific later if need be.